### PR TITLE
11.0 merge from 10.0

### PR DESCRIPTION
**Pull request by em230418**

### CODE REVIEW

File: website_event_attendee_fields/__manifest__.py [hunk 1]
```diff
@@ -39,5 +39,5 @@
     "post_init_hook": None,
 
     "auto_install": False,
-    "installable": False,
+    "installable": True,
```
Comment (Ramil-Mukhametzyanov) [inline website_event_attendee_fields/__manifest__.py]: :cactus: This module has not been ported yet.

File: website_event_attendee_signup/__manifest__.py [hunk 2]
```diff
@@ -33,5 +33,5 @@
     "post_init_hook": None,
 
     "auto_install": False,
-    "installable": False,
+    "installable": True,
```
Comment (Ramil-Mukhametzyanov) [inline website_event_attendee_signup/__manifest__.py]: :cactus: This module has not been ported yet

File: website_event_require_login/__manifest__.py [hunk 3]
```diff
@@ -31,5 +31,5 @@
     "post_init_hook": None,
 
     "auto_install": False,
-    "installable": False,
+    "installable": True,
```
Comment (Ramil-Mukhametzyanov) [inline website_event_require_login/__manifest__.py]: :cactus: This module has not been ported yet

File: website_multi_company_blog/doc/changelog.rst [hunk 4]
```diff
@@ -1,3 +1,9 @@
+`1.0.1`
+-------
+
+- **Fix:** all blogs were shown to admin regardless of website
+- **Fix:** Access error on opening some blogs/posts
+
```
[the remark this answers is not in the document]
Comment (Ramil-Mukhametzyanov) [reply website_multi_company_blog/doc/changelog.rst]: :+1: checked

**Comment by em230418** [comment]
I approve to merge it now

**Pull request merged**